src/connectedk8s/azext_connectedk8s/custom.py

```diff
@@ -1952,15 +1958,26 @@ def delete_cc_resource(
     resource_group_name: str,
     cluster_name: str,
     no_wait: bool,
+    force: bool = False,
 ) -> LROPoller[None]:
     print(f"Step: {utils.get_utctimestring()}: Deleting ARM resource")
     try:
-        poller: LROPoller[None] = sdk_no_wait(
-            no_wait,
-            client.begin_delete,
-            resource_group_name=resource_group_name,
-            cluster_name=cluster_name,
-        )
+        poller: LROPoller[None]
```

Comment:
params = {"force": True} if force else {}

poller = sdk_no_wait(
    no_wait,
    client.begin_delete,
    resource_group_name=resource_group_name,
    cluster_name=cluster_name,
    params=params
)

Reply:
had offline conversation, sdk may or may not handle None properly

Reply:
Hey, just trying to avoid passing params as none so that sdk handling need not be guessed.